packages/svelte/README.md

```diff
@@ -29,7 +29,7 @@ If you prefer [Yarn](https://yarnpkg.com/en/), use the following command
 instead:
 
 ```bash
-yarn add -D @carbon/charts-svelte d3
+yarn add -D @carbon/charts-svelte @carbon/styles d3
```

Comment:
doesn't `@carbon/styles` get installed when installing `@carbon/charts-svelte`?

`@carbon/charts` I believe is a dep in `@carbon/charts-svelte`, and `@carbon/charts` depends on `@carbon/styles`

Reply:
Also why `-D`? are these dev deps?

Reply:
Nice catch. Resolved in https://github.com/carbon-design-system/carbon-charts/commit/aed7cf633efd7917be3092e727cd3b79d71fc414#diff-85c942869817560e818cab6cfff9460e631d0ec76ca4a24cda74b337b84f7ecf

> Also why -D? are these dev deps?

Correct. For Svelte, deps can be installed as development dependencies.